packages/markers/src/browser/problem/problem-frontend-module.ts

```diff
@@ -22,10 +22,10 @@ import { FrontendApplicationContribution, bindViewContribution } from '@theia/co
 import { ProblemManager } from './problem-manager';
 import { WidgetFactory } from '@theia/core/lib/browser/widget-manager';
 import { NavigatorTreeDecorator } from '@theia/navigator/lib/browser/navigator-decorator-service';
-import { ProblemDecorator } from './problem-decorator';
+import { ProblemDecorator, ProblemTabBarDecorator } from './problem-decorator';
 import { TabBarToolbarContribution } from '@theia/core/lib/browser/shell/tab-bar-toolbar';
-
 import '../../../src/browser/style/index.css';
+import { TabBarDecorator } from '@theia/core/lib/browser/shell/tab-bar-decorator';
```

Comment:
[minor] add this import statement with other `imports` and leave the `css` import alone.